rubi/rubi/client.py

```diff
@@ -87,6 +90,9 @@ def __init__(
             network=self.network, network_tokens=self.tokens
         )
 
+        # Order tracking
+        self.active_limit_orders: Dict[int, ActiveLimitOrder] = {}
+
     @classmethod
     def from_http_node_url(
```

Comment:
on initialization, we could build the existing `active_limit_orders` for a client utilizing the `get_offers` function. however, given the need to initialize an `event_poller` this may not be the best UX, and we may be better off just having an easy function that turns the query into `ActiveLimitOrder`s